test_runner/regress/test_compute_metrics.py

```diff
@@ -1,21 +1,146 @@
-from fixtures.neon_fixtures import NeonEnv
+from __future__ import annotations
 
+import shutil
+import subprocess
+import time
+from pathlib import Path
+from typing import TYPE_CHECKING, Any, TypedDict, cast
 
-def test_compute_metrics(neon_simple_env: NeonEnv):
+import pytest
+import requests
+import yaml
+from fixtures.log_helper import log
+
+if TYPE_CHECKING:
+    from fixtures.neon_fixtures import NeonEnv
+    from pytest import MonkeyPatch
+
+    class Metric(TypedDict):
+        metric_name: str
+        type: str
+        help: str
+        key_labels: tuple[str] | None
+        values: tuple[str]
+        query: str | None
+        query_ref: str | None
+
+    class Collector(TypedDict):
+        collector_name: str
+        metrics: list[Metric]
+        queries: list[Query] | None
+
+    class Query(TypedDict):
+        query_name: str
+        query: str
+
+
+CONFIG_DIR = Path(__file__).parents[2] / "compute" / "etc"
```

Comment:
There's [`base_dir`](https://github.com/neondatabase/neon/blob/b30a4a4b577a51e875b6cd2b0172a53a199f7838/test_runner/fixtures/neon_fixtures.py#L132-L138) fixture that points to the project root
I'd use it, and convert `CONFIG_DIR` to a fixture as well (and make it an instance of `pathlib.Path`)